### PR TITLE
fix: utils path and change the typography hook and util

### PR DESCRIPTION
**Pull request by Arthurmqz**
A small fix to avoid react package issues

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/components/typography/src/utils/getResposiveTypography.ts [hunk 1]
```diff
@@ -1,8 +1,5 @@
-import { Breakpoints } from '../../react/src/theme/theme.constants';
-import {
-  TypographyStyledProps,
-  TypographyType,
-} from '../typography/src/typography.types';
+import { Breakpoints } from '../../../../react/src/theme/theme.constants';
```
Comment (joaoprcosta) [inline packages/components/typography/src/utils/getResposiveTypography.ts:1]:
```suggestion
import { Breakpoints } from '@douro-ui/react';
```

File: packages/components/typography/src/hook/__tests__/useTypography.spec.tsx [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-  getResponsiveTypography,
-} from '../../../../components/utils';
-import { useTheme } from '../useTheme';
-import { useWindowDimensions } from '../useWindowDimensions';
-import { mockTheme } from '../../../../../tests/__mocks__/theme';
-
-jest.mock('../useTheme');
-jest.mock('../useWindowDimensions');
-jest.mock('../../../../components/utils', () => ({
+import { Breakpoints } from '../../../../../react/src/theme/theme.constants';
```
Comment (joaoprcosta) [inline packages/components/typography/src/hook/__tests__/useTypography.spec.tsx:7]:
```suggestion
import { Breakpoints } from '@douro-ui/react';
```

**Comment by joaoprcosta** [comment]
Folders always have plural names because could store multiple files. Change `hook` to `hooks`

File: packages/components/typography/src/hooks/__tests__/useTypography.spec.tsx [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
-} from '../../../../components/utils';
-import { useTheme } from '../useTheme';
-import { useWindowDimensions } from '../useWindowDimensions';
-import { mockTheme } from '../../../../../tests/__mocks__/theme';
-
-jest.mock('../useTheme');
-jest.mock('../useWindowDimensions');
-jest.mock('../../../../components/utils', () => ({
+import { Breakpoints } from '@douro-ui/react';
+import { getWindowSize } from '@douro-ui/react';
```
Comment (joaoprcosta) [inline packages/components/typography/src/hooks/__tests__/useTypography.spec.tsx:8]:
```suggestion
import { Breakpoints, getWindowSize, useTheme, useWindowDimensions } from '@douro-ui/react';
```

**Review by joaoprcosta — APPROVED** [review]

**Review by vzdiogo — APPROVED** [review]

**Pull request merged**